.circleci/config.yml

```diff
@@ -23,6 +23,7 @@ jobs:
           command: |
             python -m venv env
             . env/bin/activate
+            pip install -U pip
```

Comment:
You can remove `pip` from the following line. I'd also move `setuptools` and `wheel` here, leaving `twine` only on the line below.

Reply:
My thinking was that perhaps in the future one of the other dependencies might cause the same problem, so it would be better to first update pip then use the updated pip to do the rest.
Naturally I forgot to remove the pip from the split out line :-)

Reply:
In the future we'll switch to `pyproject.toml`, so these will be updated automatically, but for now it's safe to do:
```
pip install -U pip setuptools wheel
pip install -U twine
```